iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageLevel.java

```diff
@@ -792,6 +794,23 @@ interface SwitchAdder extends IdentifiableAdder<SwitchAdder> {
          */
         Bus getBus2(String switchId);
 
+        default Collection<Bus> getBusesFromMergedBusId(String mergedBusId) {
```

Comment:
getBusesFromCalculatedBusId ?

Reply:
merge bus is not something describe on the API and for a node breaker voltage level, it is more a calculated bus.

Reply:
Yes I based the naming on the method `getMergedBus()` in BusView